src/lib/battery/battery.cpp

```diff
@@ -153,7 +153,9 @@ battery_status_s Battery::getBatteryStatus()
 {
 	battery_status_s battery_status{};
 	battery_status.voltage_v = _voltage_v;
+	battery_status.voltage_filtered_v = _ocv_filter_v.getState();
```

Comment:
I would put the measured voltage (`_voltage_v`) here, since the OCV is just an estimation and some people actually want to see the real measured value.